style.css

```diff
@@ -2,7 +2,7 @@
 Theme Name:         ALPS for WordPress
 Theme URI:          https://adventist.io/themes
 Description:        ALPS is the Adventist Living Pattern Library theme based off of the Sage WordPress theme.
-Version:            3.0.32
+Version:            3.0.33
```

Comment:
Please don't edit these in the  branch. This change will be made AFTER the  merge.